src/classes/CW_MarcomBriefViewController.cls

```diff
@@ -0,0 +1,313 @@
+		
+		languages = pageBrandHelper.languages;
+    }
+    
+    //load all th existing schedules
+    public void loadSchedules()
+    {
+    	Apexpages.currentPage().getParameters().put(ConstantUtil.PAGEPARAM_CAMPAIGNID,camp.Id);
+    	CW_ScheduleDetailsController sdc = new CW_ScheduleDetailsController();
+    	if(sdc.schedules != null && !sdc.schedules.isEmpty())
+    	{
+    		schedules = sdc.schedules;
+    		schFields = sdc.schFields;
+    	}
+    	
+    	Apexpages.currentPage().getParameters().remove(ConstantUtil.PAGEPARAM_CAMPAIGNID);
+    }
+    
+    //load all assets
+	public void initAssetPage()
+	{
+		List<WrapperUtil.selectWrapper> tempList=new List<WrapperUtil.selectWrapper>();
+		assetTypeOptions= new List<List<WrapperUtil.selectWrapper>> ();
+		
+		for (WrapperUtil.selectWrapper rec :  WrapperUtil.createWrapperList(
+							[select Asset_Type__r.Id, Asset_Type__r.Name from CW_Campaign_Asset_Type__c where Campaign__c = :camp.Id],
+							ConstantUtil.CW_CAT_REL_ASSETTYPE,ConstantUtil.SOBJECT_NAME,ConstantUtil.SOBJECT_ID,ConstantUtil.SOBJECT_ID)){
+				tempList.Add(rec);
+				if(tempList.size()>6){
+					assetTypeOptions.Add(tempList);
+					tempList=new List<WrapperUtil.selectWrapper>();
+				}
+			}
+			if(tempList.size()>0)
+					assetTypeOptions.Add(tempList);
+		assetTypeId = assetTypeOptions.isEmpty() ? null : (Id) assetTypeOptions.get(0).get(0).optionValue;
+	}
+    
+    //query all the campaign fields
+  	private CW_Campaign__c getCampaignInfo() {
+        String queryString = 'SELECT ';
+        for (Schema.SObjectField f : Schema.SObjectType.CW_Campaign__c.fields.getMap().values())
+        {
+            queryString += f.getDescribe().getName() + ', ';
+        }
+        //queryString = queryString.removeEnd(', ');
+        queryString += 'Owner.Name FROM CW_Campaign__c WHERE Id =\'' + camp.Id + '\'';
+        
+        return Database.query(queryString);
+    }
+    
+    public pageReference editRecord()
+    {
+    	PageReference ref = Page.CW_WizardStartPageSelector;
+    	ref.getParameters().put(ConstantUtil.SOBJECT_ID,camp.Id);
+    	ref.getParameters().put(ConstantUtil.PAGEPARAM_ISBRIEFVIEWMODE,'False');
+    	return ref;
+    }
+    
+    public pageReference deleteRecord()
+    {
+    	try 
+    	{
+    		delete camp;
+    	
+	    	PageReference pageRef = new PageReference('/' + Schema.sObjectType.CW_Campaign__c.keyPrefix + '/o');    
+	    	pageRef.setRedirect(true);
+	    	return pageRef; 
+	    	
+    	}catch(Exception ex){
+			ApexPages.addMessages(ex);
+		}
+		
+		return null;
+    }
+    
+    public pageReference backToListView()
+    {
+    	PageReference pageRef = new PageReference('/' + Schema.sObjectType.CW_Campaign__c.keyPrefix + '/o');    
+    	pageRef.setRedirect(true);
+    	return pageRef; 
+    }
+    
+    public List<WagerUplift> getWagerUplift() {
+            return getWagerUpliftFields();
+    }
+    
+    public List<NGRUplift> getNGRUplift() {
```

Comment:
Methods should use the `lowerCamelCase` naming convention. I suggest rename accordingly.